resources/base-runtime-mock.cfg

```diff
@@ -28,7 +28,7 @@ mdpolicy=group:primary
 
 [demorepo]
```

Comment:
The repo update works as expected. Did you also want to change name of the repo to something other than `[demorepo]`--since ends up appearing in `/etc/yum.repos.d/build.repo` within the docker image?